mjpc/planners/gradient/planner.cc

```diff
@@ -200,7 +203,8 @@ void GradientPlanner::OptimizePolicy(int horizon, ThreadPool& pool) {
     model_derivative.Compute(
         model, data_, trajectory[0].states.data(), trajectory[0].actions.data(),
         trajectory[0].times.data(), dim_state, dim_state_derivative, dim_action,
-        dim_sensor, horizon, settings.fd_tolerance, settings.fd_mode, pool);
+        dim_sensor, horizon, settings.fd_tolerance, settings.fd_mode, pool,
+        derivative_skip_);
```

Comment:
Because derivative_skip_ can change in the UI thread, maybe copy it into a local variable outside the loop, so that all the calls to model_derivative.Compute get the same value.

Reply:
Done.